### PR TITLE
[karmor summary] Added Time Stamps and Count for Ingress and Egress Connections

### PR DESCRIPTION
**Pull request by yasin-cs-ko-ak**
This is enhancement for `karmor summary`  obseravability.
- Added `Count` and `Updated Time` in these files summary.go, observability.go and kubearmor.go
- Updated `Count` and `Updated Time` in `observability.proto` under `message SysNwSummaryData` here.

Note: We have to replace this line `opb "github.com/accuknox/auto-policy-discovery/src/protobuf/v1/observability"` in [this](https://github.com/kubearmor/kubearmor-client/blob/main/summary/table.go) file with latest updated protobuf to reflect the changes that made in discovery-engine.

### CODE REVIEW

**Comment by yasin-cs-ko-ak** [comment]
@nyrahul @Ankurk99 @seswarrajan Kindly verify this PR.

File: src/conf/local-file.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 application:
   name: knoxautopolicy
   network:
-    operation-mode: 1                            # 1: cronjob | 2: one-time-job
+    operation-mode: 0                            # 1: cronjob | 2: one-time-job
```
Comment (Ankurk99) [inline src/conf/local-file.yaml]: I think you added this for testing, can you please revert it back?

File: src/protobuf/v1/config/config.pb.go [hunk 2]
```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.27.1
-// 	protoc        v3.19.3
+// 	protoc        v3.12.4
```
Comment (Ankurk99) [inline src/protobuf/v1/config/config.pb.go]: Why are we downgrading this?

**Review by Ankurk99 — CHANGES_REQUESTED** [review]
Looks good, but there seems to be a lot of noise in the PR. Can you please remove the binary files and the changes which are downgrading protoc?

File: src/conf/local-file.yaml (hunk 1 above)
Comment (seswarrajan) [inline src/conf/local-file.yaml]:
```suggestion
    operation-mode: 1                            # 1: cronjob | 2: one-time-job
```

**Review by seswarrajan — CHANGES_REQUESTED** [review]
1. Refrain from adding scripts/?_journal=OFF and scripts/accuknox.db files to the PR. 
2. Do not modify conf/local-file.yaml unless necessary. 
3. Do not add all protobuf compiled files. Add only the files with respective changes. 

Rest LGTM

File: src/conf/local-file.yaml (hunk 1 above)
Comment (yasin-cs-ko-ak) [reply src/conf/local-file.yaml]: Reverted

File: src/protobuf/v1/config/config.pb.go (hunk 2 above)
Comment (yasin-cs-ko-ak) [reply src/protobuf/v1/config/config.pb.go]: Reverted to 3.15.8 existing version.

File: src/conf/local-file.yaml (hunk 1 above)
Comment (yasin-cs-ko-ak) [reply src/conf/local-file.yaml]: Changes are done.

**Comment by seswarrajan** [comment]
image

Do not add all protobuf changes to the PR. 
I think the change is done only for observability. Hence only add files related to observability and ignore other files.

**Review by seswarrajan — APPROVED** [review]
LGTM

**Comment by Ankurk99** [comment]
LGTM :rocket: great work @yasin-cs-ko-ak

**Review by Ankurk99 — APPROVED** [review]

**Pull request merged**